### ISSUE TITLE
Recursive 'itemstacks' definition

### ISSUE THREAD

**Issue by barneygale** [PrismarineJS/minecraft-data#359]
The definition of PE's 'itemstacks' type itself refers to 'itemstacks': https://github.com/PrismarineJS/minecraft-data/blob/4e9c6056ccb42379f6ff4c017504f26441edf992/data/pe/0.15/protocol.json#L316-L330

**Comment by rom1504** [comment]
Yeah that seems broken indeed.
I believe @extremeheat is working on an updated version for a newer version

**Comment by barneygale** [comment]
Nice. I'm going to try to put together a definition for RakNet too, which shouldn't ever need to change. Don't expect it any time soon, though :)

**Comment by rom1504** [comment]
We have one at https://github.com/mhsjlw/node-raknet/blob/master/data/protocol.json
It works, just missing implementing ack queues on the js side

**Comment by extremeheat** [comment]
Yes, I'm working on updating it with the protocol library to 1.16. What's your use case? I should have something working soon.

**Comment by extremeheat** [comment]
Here's an updated ProtoDef protocol definition for bedrock 1.16 if you needed, https://github.com/extremeheat/pocket-minecraft-protocol/blob/new/data/newproto.json

I have it generated from two yaml files for now, https://github.com/extremeheat/pocket-minecraft-protocol/tree/new/data/new through this tool https://github.com/extremeheat/protodef-yaml. This makes maintenance easier and adds comment support.

Expect lots of errors and changes, most of this is manual and has yet to be tested. Once it's stable we can move the definitions to mc-data.

**Comment by barneygale** [comment]
Amazing! I'll give it a spin soon.
